### DISCUSSION TITLE
Retrieves the current canvas image as a Uint8List

### DISCUSSION THREAD

**Discussion by iOSyan** [hm21/pro_image_editor#82] [Q&A] [answered]
Hello, can you make it so that clicking a custom button retrieves the current canvas image as a Uint8List?
Because our project needs to save templates and be able to call them next time, I need to record the current image of the template.

**Comment by hm21** [comment, marked as answer]
If you want to take a screenshot without closing the editor, you can do so as follows `editorKey.currentState!.captureEditorImage();`
